root/static/html/admin.html

```diff
@@ -0,0 +1,198 @@
+<div class="container">
```

Comment:
I'd rather have separate templates for repo/repos/user/users. If they share piece of code, that can go into a separate file to be included from these 4.